### PR TITLE
first draft of the comments ui

### PR DESCRIPTION
**Pull request by kwasniew**
## About the changes

Screenshot 2022-11-15 at 17 32 21

What it adds?
* list of comments
* adding a comment (can only add comment when the field is not empty)
* banner zIndex fix

### CODE REVIEW

File: frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestOverview.tsx [hunk 1]
```diff
@@ -109,6 +133,15 @@ export const ChangeRequestOverview: FC = () => {
                     <StyledInnerContainer>
                         Changes
                         <ChangeRequest changeRequest={changeRequest} />
+                        {changeRequest.comments.map(comment => (
```
Comment (Tymek) [inline frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestOverview.tsx:136]:
```suggestion
                        {changeRequest.comments?.map(comment => (
```

File: frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestOverview.tsx [hunk 2]
```diff
@@ -109,6 +133,18 @@ export const ChangeRequestOverview: FC = () => {
                     <StyledInnerContainer>
                         Changes
                         <ChangeRequest changeRequest={changeRequest} />
+                        {changeRequest.comments?.map(comment => (
+                            <ChangeRequestComment
+                                key={comment.id}
+                                comment={comment}
+                            />
+                        ))}
+                        <AddComment
```
Comment (Tymek) [inline frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestOverview.tsx:142]: `AddComment` could be `AddCommentField`

**Review by Tymek — APPROVED** [review]
👍🏻

File: frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestOverview.tsx (hunk 2 above)
Comment (kwasniew) [reply frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestOverview.tsx]: Refactoring

**Pull request merged**